shim/src/sanitize/accessors.js

```diff
@@ -0,0 +1,62 @@
+// Adapted from SES/Caja - Copyright (C) 2011 Google Inc.
+// https://github.com/google/caja/blob/master/src/com/google/caja/ses/startSES.js
+// https://github.com/google/caja/blob/master/src/com/google/caja/ses/repairES5.js
```

Comment:
We should give credit to the original implementation, even if this is small subset and modernized version of the code the code. Doing so will also help with continuity.